source/entities/Totem.hx

```diff
@@ -14,15 +14,18 @@ using flixel.util.FlxSpriteUtil;
 class Totem extends ISelectable {
 	var totem:FlxSprite;
 
+	var bullets:FlxGroup;
+
 	var shotSpeed:Float = 1;
 	var shotTimer:Float = 1;
 
 	private var _range:FlxSprite;
 
 	private var distortion:Float = .5;
 
-	public function new(shotSpeed:Float) {
+	public function new(shotSpeed:Float, bulletGroup:FlxGroup) {
```

Comment:
Instead of using the `FlxG.state`, we can explicitly pass in the group that way in case we want to make multiple things, we can give multiple groups to the producer.